client/my-sites/plugins/style.scss

```diff
@@ -359,6 +356,12 @@ $plugin-details-header-padding: 100px;
 			border-bottom: 1px solid var( --studio-gray-5 );
 		}
 	}
+
+	@media screen and ( max-width: 480px ) {
+		.section-nav-tab:first-child .section-nav-tab__link {
+			padding-left: 15px;
+		}
+	}
```

Comment:
instead of creating an override of an override, can set `padding-left:0` only for `min-width: 480px?

https://github.com/Automattic/wp-calypso/blob/9a90af6eaeee37c2e7650e83386dd2a54b7f48c1/client/my-sites/plugins/style.scss#L298-L300

Reply:
Agreed, implemented at 67c7b2a83148bbf5b1669f2e0a892834b1e670e3 with `min-width: 481` as min is exclusive.